src/components/App.tsx

```diff
@@ -666,13 +661,15 @@ export class App extends React.Component<AppProps, AppState> {
     </Split>;
 
     return <div className="fill">
-      {this.state.toast &&
+      <div className="toast-container">
```

Comment:
This feels like it shouldn't belong in the App component, it's an internal detail of how toasts are displayed. Perhaps you need a `ToastContainer` component that accepts `Toast` elements as children.